src/main.c

```diff
@@ -238,6 +239,7 @@ CF_ABS("auth_query", CF_STR, cf_auth_query, 0, "SELECT usename, passwd FROM pg_s
 CF_ABS("auth_type", CF_LOOKUP(auth_type_map), cf_auth_type, 0, "md5"),
 CF_ABS("auth_user", CF_STR, cf_auth_user, 0, NULL),
 CF_ABS("autodb_idle_timeout", CF_TIME_USEC, cf_autodb_idle_timeout, 0, "3600"),
+CF_ABS("track_startup_parameters", CF_STR, cf_track_startup_parameters, 0, NULL),
```

Comment:
If this setting is not part of the config pgbouncer now segfaults (which is why all tests in CI are failing). Changing default to `""` seems to help.
```suggestion
CF_ABS("track_startup_parameters", CF_STR, cf_track_startup_parameters, 0, ""),
```